### PR TITLE
set currentColor as default color

### LINKED ISSUE #556: Implement currentColor in SVG
It would be useful to stroke elements in SVG images with `currentColor` rather than `#000000`.

Info about currentColor:

https://developer.mozilla.org/en/docs/Web/CSS/color_value#currentcolor_keyword
https://css-tricks.com/cascading-svg-fill-color

Example CSS:

```css
<style>
        svg g.note {
                color: red;
                stroke: deeppink !important;
                fill: blue;
        }
</style>
```

Current SVG output from verovio uses a stroke of `#000000` for stems:

```xml
      <g class="note" id="note-L2F1">
        <use xlink:href="#E0A4" x="1061" y="1940" height="720px" width="720px" />
        <g class="stem" id="stem-1">
         <path d="M1277 1918 L1277 1310" stroke="#000000" stroke-width="20" />
         <g class="flag" id="flag-1">
          <use xlink:href="#E240" x="1267" y="1310" height="720px" width="720px" />
         </g>
        </g>
```

This will be rendered as:

screen shot 2017-04-22 at 8 19 41 am

The stem remains black, even if `!important` is added to try to override the black color:
https://css-tricks.com/when-using-important-is-the-right-choice

However, if the stroke is set to `currentColor`, CSS styling can be applied:

```xml
      <g class="note" id="note-L2F1">
        <use xlink:href="#E0A4" x="1061" y="1940" height="720px" width="720px" />
        <g class="stem" id="stem-1">
         <path d="M1277 1918 L1277 1310" stroke="currentColor" stroke-width="20" />
         <g class="flag" id="flag-1">
          <use xlink:href="#E240" x="1267" y="1310" height="720px" width="720px" />
         </g>
        </g>
```

which is rendered as:

screen shot 2017-04-22 at 8 21 54 am

In this case the stem is colored red from the CSS `color` parameter setting (not `stroke` since that is hard-coded to the soft variable `currentColor`).

Also if `fill="#000000` is ever used (it is not in my test image), then it should be replaced with `fill=currentColor` unless black is explicitly wanted such as from an MEI `color="#000000"` parameter.

Also note that the notehead should be placed after the stem element in SVG to make the stem go behind the notehead (I think that is already a reported issue).

In other words:
```xml
       <g class="note" id="note-L2F1">
        <g class="stem" id="stem-0000000768152788">
         <path d="M1277 1918 L1277 1310" stroke="currentColor" stroke-width="20" />
         <g class="flag" id="flag-0000001819705799">
          <use xlink:href="#E240" x="1267" y="1310" height="720px" width="720px" />
         </g>
        </g>
        <use xlink:href="#E0A4" x="1061" y="1940" height="720px" width="720px" />
       </g>
```

Which renders as:

screen shot 2017-04-22 at 8 25 56 am

### PR DESCRIPTION
**Pull request by rettinghaus**
fixes #556
(after two years all original problems are gone, so now it's time to finally address it)

### CODE REVIEW

File: Verovio.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -2637,10 +2637,9 @@
 			};
 			buildConfigurationList = 8F086EA3188534680037FD8E /* Build configuration list for PBXProject "Verovio" */;
 			compatibilityVersion = "Xcode 3.2";
-			developmentRegion = English;
```
Comment (lpugin) [inline Verovio.xcodeproj/project.pbxproj]: Can you discard the changes on this file?

Comment (rettinghaus) [reply Verovio.xcodeproj/project.pbxproj]: X-Code wanted this change, which seems understandable to me. 
You may check this: 
https://stackoverflow.com/questions/7051120/why-doesnt-my-file-move-into-en-lproj-but-instead-into-a-new-english-lproj

But if you still want it to be reverted, I'll do so.

Comment (lpugin) [reply Verovio.xcodeproj/project.pbxproj]: > https://stackoverflow.com/questions/7051120/why-doesnt-my-file-move-into-en-lproj-but-instead-into-a-new-english-lproj

This is from 8 years ago... What version of XCode are you using? I am on 10.1

Comment (craigsapp) [reply Verovio.xcodeproj/project.pbxproj]: I don't know if it is identical in purpose, but there is a environmental variable called `$MACOSX_DEPLOYMENT_TARGET` that specifies the oldest MacOS operating system that the code should be compiled to work on (such as "10.4" to run on computers as old as MacOS 10.4).  If the variable is not set, then only the version of the MacOS on the compiling computer and later can be used to run the compiled program.

**Comment by rettinghaus** [comment]
I‘m using the latest version of X-Code. 
And they recommended eight years ago already to use ISO codes.

**Pull request merged**